packages/packages.csv

```diff
@@ -173,6 +171,7 @@ Microsoft.Extensions.TaskCache.Sources,noship,exclude
 Microsoft.Extensions.TypeNameHelper.Sources,noship,exclude
 Microsoft.Extensions.WebEncoders,ship,include
 Microsoft.Extensions.WebEncoders.Sources,noship,exclude
+Microsoft.Extensions.WebSockets.Internal,noship,exclude
```

Comment:
By ship, we mean publish to nuget.org when 1.2.0 comes out. Does that sound right? If so, can we come up with a better name for this or make it a `.Sources` package?